src/details.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-underscore-dangle */
+import { getDetails } from './tvmaze-api';
+import './images/loading.svg';
+import { addComment, getCommentList } from './Involvement-api';
+
+export async function commentsCount(promise) {
```

Comment:
- [ ] [OPTIONAL] we can work on this function so the comments count can be updating in real time, instead of refreshing the page. I have made this request optional because I know that you did the work alone and it's not easy.